### ISSUE TITLE
Change the overall content theme from Harry Potter. #29

### ISSUE THREAD

**Issue by NathanS1** [rsanchez-wsu/RaiderPlanner#29]
Currently, the application is Harry Potter themed. This doesn't fit the Raider Planner style that we are going for. I suggests changing all of the school classes, class requirements, teachers names and info, etc... Coursework and teachers could be taken from Wright State curriculum. This could address what it takes to earn a BS or BA in Computer Science or Computer Engineering. Real names of classes and teachers could be used. The scope of what gets changed or how many classes that are added should be addressed before starting.

**Comment by LoganRickert** [comment]
I think that we probably shouldn't use real names for teachers.

**Comment by Alienturnedhuman** [comment]
Hi (member of original PearPlanner team here), just to clear up any misunderstandings regarding this. The xml files are not part of the application - they are tailored study profiles, unique for each student, as different students will have different timetables and project deadlines depending on what they are studying, who their supervisors/tutors are etc... The PearPlanner / RaiderPlanner is then a tool for them to organise their workflow, so the file needs to be unique for them.

Obviously, there wasn't a need for us to create hundreds of unique different study profiles, just two different ones were enough to demonstrate the functionality - so I doubt that there is any need for you to generate hundreds either for yours - however I just wanted to clear up any ambiguity as this thread implied the idea that the study profile was part of the application, and therefore standardized for each user.

When we made the original software package, we included the Harry Potter xml as an easter egg, but also did one based on our own timetable. There are also study profile update files, that students can load in the event their timetable gets changed.

Note, our original project specification specifically did not permit for a network based solution for this (ie, the software connecting to a remote server to download the latest timetable information) and we had to develop a system that worked on supplied files. Looking at the direction this project is now taking (ie, web interface) that part of the specification may now be redundant.

**Comment by rsanchez-wsu** [comment]
@NathanS1 Bear in mind that there are two issues already open related to the application theme (#5 and #16).  Consider whether what you have summarized in this issue is already captured in one of those or whether this constitutes a sub-issue of one or both of those.

@Alienturnedhuman Thanks very much for the additional information.  One does not always have the benefit of a project's original author being around to help inform discussions.  I appreciate you taking the time.

**Comment by Alienturnedhuman** [comment]
@rsanchez-wsu No problem, I'm interested it see where the project ends up!

Just so there is no further ambiguity, the "Harry Potter" stuff is not a 'theme' - in this regard the software is themeless. It is a piece of software that any school or college can generate study planner XML files for. The XML files are released one for each semester, with and corrections/amendments/updates to that semester being provided with a semester update XML file 

There are examples of all of these XML files in the StudyProfiles folder:

https://github.com/rsanchez-wsu/RaiderPlanner/tree/master/StudyProfiles

You can load one by clicking "Import HUB File" in the GUI. If there is not a study profile loaded for that year/semester no then it will load it into the profile. I don't think there is the option to overwrite an existing year/semester - so this may be a feature implementation you may wish to consider, but obviously with a user prompt "There already exists a study profile for this year / semester do you wish to overwrite" - you may also consider upgrading the Study Profile XML file to include the Student's ID number, as a way of verifying that a student is adding a study profile meant for them.

There is a demonstration of reseting the software to blank (no study profiles) by moving/deleting StudyProfile.dat and then loading a study profile in this YouTube Video (start from 4:25)

https://www.youtube.com/watch?v=R5fcENosJbA&t=4m25s

Then at 11:25 I then load a Study Profile update file (one of the updates causes the name of the professor to be changed, who I circle with the mouse pointer)
